### PR TITLE
Update Doc for Filters 

### PR DESCRIPTION
**Pull request by sowu880**
Give more detailed descriptions and examples for our date and datetime related filters

### CODE REVIEW

File: docs/Filters-and-Tags.md [hunk 1]
```diff
@@ -55,10 +55,44 @@ If these filters do not meet your needs, you can also write your own filters.
...
+> [Note] : Input string will not affect the result.
+- add_seconds
+```
+    {{ "1970-01-01T00:01:00.000+10:00" | add_seconds: -60, 'utc' }} -> 1969-12-31T14:00:00.000Z
+    {{ "1970-01-01T00:01:00Z" | add_seconds: 60.1230 }} -> 1970-01-01T00:02:00.123Z
+    {{ "1970-01-01T00:01:00+14:00" | add_seconds: 60, 'preserve' }} -> 1970-01-01T00:01:01+14:00
+```
+
+> [Note] : If the input is a partial datetime without time zone, it will be set to the first day of the year and 00:00:00 clock time with local time zone as suffix. e.g. In the location with +08:00 time zone, the input string "201101031434" will be filled to "20110103143400+0800",
+   the template `{{ "201101031434"| format_as_date_time: 'utc'}}` will output 2011-01-03T06:34:00Z at +08:00 location.
```
Comment (BoyaWu10) [inline docs/Filters-and-Tags.md:94]: Nit: the template `{{ "201101031434" | format_as_date_time: 'utc'}}` will output `2011-01-03T06:34:00Z` when running at +08:00 location.

File: docs/Filters-and-Tags.md [hunk 2]
```diff
@@ -55,10 +55,43 @@ If these filters do not meet your needs, you can also write your own filters.
...
 | now | Provides current time in a specific format. The default format is "yyyy-MM-ddTHH:mm:ss.FFFZ". | `{{ '' \| now: 'dddd, dd MMMM yyyy HH:mm:ss' }}` |
-| add_seconds | Adds double seconds on datetime in FHIR format. Provides parameters to set time zones: `preserve`, `utc`, `local`. The default is set to the parameter `preserve`. | `{{ "2021-01-01T00:00:00Z" \| add_seconds:100.1 }}` |
+| add_seconds | Adds double seconds on valid [FHIR datetime](http://hl7.org/fhir/R4/datatypes.html) as input (e.g.2021-01-01T00:00:00Z ) and output a valid FHIR datetime. Provides parameters of double seconds to add and settings of time zones: `preserve`, `utc`, `local`. The default set of time zone parameter is `preserve`. | `{{ "2021-01-01T00:00:00Z" \| add_seconds:100.1, 'utc' }}` |
```
Comment (ginalee-dotcom) [inline docs/Filters-and-Tags.md:61]: Adds double seconds on a valid [FHIR datetime](http://hl7.org/fhir/R4/datatypes.html) (e.g. 2021-01-01T00:00:00Z). Provides parameters to handle different time zones: `preserve`, `utc`, `local`. The default method is `preserve`.

File: docs/Filters-and-Tags.md [hunk 3]
```diff
@@ -55,10 +55,43 @@ If these filters do not meet your needs, you can also write your own filters.
...
+
+Examples:
```
Comment (ginalee-dotcom) [inline docs/Filters-and-Tags.md:63]: DateTime filter usage examples:

**Review by ginalee-dotcom — DISMISSED** [review]

**Review by BoyaWu10 — APPROVED** [review]

**Review by qiwjin — APPROVED** [review]

**Comment by yankunhuang-pku** [comment]
@sowu880 Are we ready to merge this PR?

**Pull request merged**